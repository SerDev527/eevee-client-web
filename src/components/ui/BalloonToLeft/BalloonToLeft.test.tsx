import React from 'react';
import renderer from 'react-test-renderer';

import { BalloonToLeft } from '.';

const toJSON = (component: renderer.ReactTestRenderer) => {
  const result = component.toJSON();
  expect(result).toBeDefined();
  expect(result).not.toBeInstanceOf(Array);
  return result as renderer.ReactTestRendererJSON;
};

test('render BalloonToLeft', () => {
  const component = renderer.create(
    <BalloonToLeft>
      <></>
    </BalloonToLeft>,
  );
  const tree = toJSON(component);
  expect(tree).toMatchSnapshot();
});
