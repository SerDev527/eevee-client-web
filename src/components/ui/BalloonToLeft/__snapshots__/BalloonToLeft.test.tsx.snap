// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`render BalloonToLeft 1`] = `
<div
  className="_balloon_8e700f"
/>
`;
